### DISCUSSION TITLE
Questions about polymer-solvent and polymer-polymer interactions

### DISCUSSION THREAD

**Discussion by rswf** [marrink-lab/polyply_1.0#391] [Q&A]
Hello !@fgrunewald 
Thank you very much for your work on polymer construction and testing in your paper, it is very meaningful and has inspired me a lot. But I still have some questions about the polymer-solvent interaction parameters after reading your paper.

The first one is the calculation of χ. According to the formula in your paper, χ is calculated from the Hansen solubility parameter of polymer-solvent. 
![{518E32A6-550F-4022-A936-6C4DB530E69C}](https://github.com/user-attachments/assets/97034d8f-044d-4e91-afa7-5d2f916cd17e)
My research topic is swelling of block copolymers and I would like to capture the polymer-solvent interaction more accurately. I am confused how the solubility parameter of the coarse-grained model is decomposed into δd, δp, δhb. I calculated the PS from the cohesive energy density with 100 repetitive units,the solubility parameter is 12.53(J/cm^3)^0.5, and the solubility parameter of ethanol (SN6r) is 14.39(J/cm^3)^0.5, both of which are below the experimental value. The χ value of 0.1288 obtained by  the formula is also less than the theoretical value. May I ask how the interaction parameters of PS and ethanol were calculated in your paper?
![{65795582-C9A8-4329-8B29-BDFABE136875}](https://github.com/user-attachments/assets/a17664b6-9a12-4645-a4e4-139a3ee665ac)
![{891F40FA-A547-41EB-86D2-C5F881546999}](https://github.com/user-attachments/assets/d95e1c78-c859-4776-bd11-873ca7876c5d) ![{5375E422-4416-43D9-809F-7AC2A29DDD51}](https://github.com/user-attachments/assets/a7db6635-78a7-4a94-be61-6a99ba1c9844) ![{1955A201-A12D-4339-8EAE-59018470A3F3}](https://github.com/user-attachments/assets/77ed876a-36da-4bc4-8c9a-b1efe5879e2f)

In addition, I noticed that the PS model I constructed also has good kinematics at 300 K, 1 bar, and the conformation of the polymer chains does not seem to be constrained (melt-like), whereas the atoms of PS simply vibrate in situ in the all-atom simulation. I have also tested the PS model of Martini3 from the polyply library and the same phenomenon exists, which is somewhat counter-intuitive. Is this normal and does it affect the behavior of PS in solvents?

**Comment by fgrunewald** [comment]
Hi @rswf, 

We do not compute $\chi$ for the Martini model. In the specific section we only compare the solubility observed in the simulation with the prediction of Hansen theory due to a lack of experimental data that is well verified. 

Secondly, computing $\chi$ via the cohesive energy is possible, but gives you so called Hildebrand Solubility parameters (i.e. $\delta$). While theoretically more simple, Hildebrand solubility theory has very poor predictive power with an accuracy of at best 50%. A good explanation of the issues is given in the Handbook by Hansen (https://www.hansen-solubility.com/Charles-Hansen/thehandbook.php). 

In addition, I do not expect Martini to give any sensible $\chi$ values obtained via CED, because Martini does not aim to reproduce the cohesive energy. In general the heat of vaporization in Martini is different from the experimental one, because Martini is parametrized on (free) energy differences. This [paper](https://pubs.acs.org/doi/10.1021/acs.jctc.9b00473) gives more background (see Figure 4 for instance). In the Martini3 parametrization paper the matter is also discussed in more detail. 

I'm not sure what `not seem to be constrained (melt-like),` means. Could you be more specific about what bothers you with the melt?

**Reply by rswf** [reply]
It looks like I misunderstood the solubility part of your paper, it should be comparing the Martini3 simulation results with the Hansen solubility parameter calculations, thanks for your reply, it cleared up my confusion.

I constructed 100 PS chains with 100 repeating units for NPT systematic simulations at 1 bar, 300 K. One of the chains is shown in the video.
https://github.com/user-attachments/assets/eeb0c7ae-7497-442f-80ab-cb044f408b06

**Reply by fgrunewald** [reply]
Is this a Martini simulation? Martini is always above the glass transition temperature even at 300K, so you would expect more movement. In general I don't think it has anything todo with polyply coordinates and more with the force field. Also note that there are no entanglements being generated in the way you run the program. However, for such short chains there are few anyways

**Reply by rswf** [reply]
Thank you for your reply. I am sorry that I still don't know enough about the Martini force field.

The video is indeed a Martini simulation which shows only the backbone of a chain. Regarding the fact that Martini is always above the glass transition temperature at 300K, I would like to know what factors are responsible for this, perhaps the reduced friction between the models after coarse graining?

My research topic involves the swelling of PS in the glassy state, which causes PS to deform plastically. Is it possible to implement a glassy state simulation of PS in Martini? Setting lower temperatures independently of the PS in the simulation to limit the movement of the PS, or assigning the PS bead types with stronger interactions?

**Reply by fgrunewald** [reply]
There are a number of factors contributing to this phenomenon, but a smoothed energy landscape is one factor. In general, we view this as an advantage, because sampling is faster and we are typically not interested in non-equilibrium processes like swelling. 

In principle, you could set a different temperature, but then you'd get energy flow from one group to another within the simulation. If you lower the temperature beyond the glassy transition it is likely your solvent would freeze. In my opinion Martini might not be your best choice or for sure easiest choice here. 

If you are only interested in local effects, all-atom simulations might still be an option.

**Reply by rswf** [reply]
Thank you very much for the key information, I know more about Martini. Looking at the parameterization process, the thermodynamic results are what Martini cares more about. Despite fitting some structural properties, Martini is still less accurate in terms of kinetics, right?

I am still interested in looking at non-equilibrium processes like block copolymer swelling on larger scales, and all-atom simulations are not sufficient.

I do appreciate you taking the time to answer my questions, even though they may be very simple.

**Reply by fgrunewald** [reply]
No worries! You're welcome to ask questions. Yes, kinetics is something that is typically not well reproduced.

**Reply by rswf** [reply]
Given the existing work, I intend to continue my research under the Martini framework. Although the new model can get the right type of beads from the octanol-water transfer free energy, there may be a lack of metrics in validating the new model to interact with the existing models. I have another question, can the transfer free energy of a molecule of substance A from substance A to substance B be used as a metric for AB interactions?
